packages/infra/config/aws/ecs.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * 512 (0.5 GB), 1024 (1 GB), 2048 (2 GB) - Available cpu values: 256 (.25 vCPU)
```

Comment:
providing a link to where this info is coming from could be useful for reference

Reply:
updated